tools/bridge/bridge/confirmation_sender.py

```diff
@@ -19,6 +20,23 @@
 logger = logging.getLogger(__name__)
 
 
+def make_sanity_check_transfer(foreign_bridge_contract_address):
+    """return a function that checks the final transfer right before it is confirmed
+
+    This is the last safety net. We do check that the transfer has
+    been sent to the foreign_bridge_contract_address here.
+    """
+    assert is_checksum_address(foreign_bridge_contract_address)
+
+    def sanity_check_transfer(transfer_event):
+        if not isinstance(transfer_event, AttributeDict):
+            raise ValueError("not an AttributeDict")
+        if transfer_event.args["to"] != foreign_bridge_contract_address:
```

Comment:
maybe better to use `is_same_address` from `eth_utils`, so that we don't accidentally think two addresses are different, even though they are just differently encoded (hex, checksum, bytes).

Reply:
I did think about that, but decided against it. That's why there's the assert is_checksum_address().

Reply:
I don't care enough to change it. If you want to have it changed, please open a PR.